components/callbacks/config.go

```diff
@@ -66,3 +68,61 @@ func ConfigProvider(dc *dynamicconfig.Collection) *Config {
 		},
 	}
 }
+
+var EndpointConfigs = dynamicconfig.NewNamespaceTypedSettingWithConverter(
+	"component.callbacks.endpointConfigs",
+	endpointConfigConverter,
+	[]EndpointConfig(nil),
+	`The per-namespace list of endpoints that are allowed for callbacks and options to use when making callback requests.
+Default is no configs, meaning all callbacks will be rejected. Any invalid configs are ignored.
+Each entry is a map with possible entries:
+	"EndpointPattern":string - (required) the host:port pattern this config applies to; * wildcards are supported
```

Comment:
```suggestion
	"Pattern":string - (required) the host:port pattern this config applies to; * wildcards are supported
```

Reply:
And clarify what the semantics of wildcards are please.
E.g. I think it's worth giving examples such as:
```
* matches everything
prefix.*.domain matches prefix.a.domain as well as prefix.a.b.domain
```

And also please explain that these rules are evaluated in the order and it only takes a single match to pass validation.